public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme

```diff
@@ -506,6 +506,11 @@ function hdbt_subtheme_form_alter(&$form, FormStateInterface $form_state, $form_
   if ($form_id == 'openid_connect_login_form') {
     $form['openid_connect_client_tunnistamo_login']['#value'] = t('Log in');
   }
+
+  // Attach the restricted-datepicker library to the "liikunta_tapahtuma" form.
+  if (str_contains($form_id, 'liikunta_tapahtuma')) {
```

Comment:
Onko `str_contains`in käyttöön jotain etuja verrattuna perinteiseen yhtäsuuruusvertailuun jota muutoin on käytetty vastaavissa yhteyksissä?

Reply:
> Onko `str_contains`in käyttöön jotain etuja verrattuna perinteiseen yhtäsuuruusvertailuun jota muutoin on käytetty vastaavissa yhteyksissä?

Lomakkeen ID voi olla `webform_submission_liikunta_tapahtuma_edit_form` tai `webform_submission_liikunta_tapahtuma_add_form`, ja näinollen `str_contains` on mielestäni "siistimpi" ratkaisu. Vaihdoin tosin tuon tarkastettavan merkkijonon nyt olemaan `webform_submission_liikunta_tapahtuma`.